Assets/Scripts/Player.cs

```diff
@@ -5,6 +5,8 @@ public class Player : Entity
 {
     private void FixedUpdate()
     {
+        if (health <= 0)
+            Debug.Log("Game Over");
```

Comment:
Let's move these lines to the `OnCollision` script, within the `if` statement in `OnCollisionEnter2D()`. We want to avoid putting something like this in an `Update` function as the health only needs to be checked when the player takes damage, so it doesn't needlessly have to check the health every frame.

Reply:
Done